lib/Basic/Platform.cpp

```diff
@@ -181,11 +181,14 @@ StringRef swift::getPlatformNameForTriple(const llvm::Triple &triple) {
   case llvm::Triple::Ananas:
   case llvm::Triple::CloudABI:
   case llvm::Triple::DragonFly:
+  case llvm::Triple::DriverKit:
```

Comment:
@egorzhdan I assume we do actually want to handle this?

Reply:
Yes, I am planning to do that. I've started upstreaming DriverKit support in the compilers, so far only the LLVM part of it has landed. We would likely need the Clang part to land first before handling this.

Reply:
Thanks, so this is fine for now?

Reply:
Yes, this is fine for now 👍